src/Views/Games/2048/Instructions.js

```diff
@@ -14,7 +14,17 @@ export default () => {
 
     return (
         <div className={clsx(classes.root)}>
-
+        <p>
+        The goal of 2048 is to create a tile with the value 2048, which is done by combining tiles that have
```

Comment:
Look at the Material-UI Typography component. Any material ui component will inherently consume the global theme. Use the paragraph prop.